x-pack/plugins/apm/public/components/app/Main/Home.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { EuiSpacer, EuiTabbedContent } from '@elastic/eui';
+import React from 'react';
+import { KueryBar } from '../../shared/KueryBar';
+import { SetupInstructionsLink } from '../../shared/SetupInstructionsLink';
+import { HeaderContainer } from '../../shared/UIComponents';
+import ServiceOverview from '../ServiceOverview';
+import TraceOverview from '../TraceOverview';
+
+export function Home() {
+  return (
+    <div>
+      <HeaderContainer>
+        <h1>APM</h1>
+        <SetupInstructionsLink />
+      </HeaderContainer>
+      <KueryBar />
+      <EuiSpacer />
+      <EuiTabbedContent
```

Comment:
The tab bar items look smaller than i.e. when you're on a specific service. Maybe we need to specify to use `m` or the size prop that matches?

Reply:
Looks like "m" is the default here and the only options seem to be "s" or "m" (need to check with EUI team on why the sizes are limited there).

The difference between this and other tabs is because we aren't using EUI for the "TabNavigation" ... we should probably switch that over to use EUI and then ask them for a bigger size option. What do you think we should do for now?

Reply:
Strange, the size is definitely smaller than when looking at the [Eui Docs](https://elastic.github.io/eui/#/navigation/tabs) if `m` is the default. Most users probably won't notice the difference between our current and the Eui tabs, unless of course we can't change to a bigger size 😉

Reply:
OK so 'm' is the default, and the only options are 's' or "not small" basically. https://github.com/elastic/eui/blob/master/src/components/tabs/tabs.js

The reason we're seeing a difference between the EUI docs and our app is because the tabs aren't actually sized directly unless they are "s", using a CSS class, otherwise they just inherit whatever font size comes from the surrounding app, in our case that's 14px. In the docs that's 16px. :/

Reply:
Would be nice to have everything in EUI use em or rem so that the ratios all stay the same if we set our default body size...

Reply:
Interestingly: when I change the prop to "s", I see the new class applied: `.euiTabs--small` ... but there appear to be no styles attached to that class in the dev tools, so the font size remains unchanged. I'll see if the EUI team has thoughts on this.

Reply:
Talked to the EUI team and this is K6 theme related. Until that theme goes away, best solution here was to add a class and override the tab size via global CSS 😷

Reply:
@jasonrhodes Are we able to use the existing tabs we have in i.e. Transactions overview page?

Reply:
Isn't it better to replace our own tabs with EUI's and then wait for them to be updated? If it's really bad we can do the global css hack.

Slightly related: we have our own styles for containers, headers and tabs several places, and it would be great to get rid of (if possible)
https://github.com/elastic/kibana/blob/889e9bc960f8ebe2d27a889f39da2567f8daf10c/x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.js#L37-L59

Reply:
> I just wanted to avoid the difference moving through multiple pages, so whichever was the most easy to implement.

Agree, there shouldn't be different tabs. I don't think it would be much harder to make them all EUI tabs.

> With regards to the container and header styles, I think we'll update them in due time when we decide to make a bigger switch to Eui. I reckon we have bigger tasks on our hands at the moment than going through the styles?

True